web/angular-wallet/src/app/main/transactions/transaction.service.ts

```diff
@@ -54,4 +67,8 @@ export class TransactionService {
         const senderPrivateKey = decryptAES(keys.signPrivateKey, hashSHA256(pin));
         return this.transactionApi.sendMoney(transaction, keys.publicKey, senderPrivateKey, recipientAddress);
     }
+    public async sendMoneyMultiOut({ transaction, pin, keys, sameAmount }: SendMoneyMultiOutRequest) {
```

Comment:
We should always return a type in TS

Reply:
```suggestion
    public async sendMoneyMultiOut({ transaction, pin, keys, sameAmount }: SendMoneyMultiOutRequest) : Promise<TransactionId> {
```

Reply:
I think it's actually inferred because this simply wraps another method.